src/mlpack/tests/load_save_test.cpp

```diff
@@ -163,10 +164,13 @@ TEST_CASE("LoadSparseTSVTest", "[LoadSaveTest]")
   // Remove the file.
   remove("test_sparse_file.tsv");
 }
+*/
```

Comment:
Any reason this is commented out?  (I see some other tests are commented out too.)

Reply:
Yes, we did not handle the sparse matrices yet. We would like to create a one load function and handle both of them